Taxopark/Taxopark/obj/project.nuget.cache

```diff
@@ -1,8 +1,28 @@
 {
```

Comment:
This file should be excluded from commit

Reply:
This file, as well as others autogenerated by VS or Docker, was removed.